### PR TITLE
COMMERCE-11068

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-11068

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccelerators.testcase [hunk 1]
```diff
@@ -244,6 +244,40 @@ definition {
 			value1 = "$ 18.90");
 	}
 
+	@description = "This is a functional test for COMMERCE-11068 - Can change the current theme to a Minium theme"
+	@priority = 4
+	test CanChangeThemeToMiniumTheme {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccelerators.testcase:249]:
```suggestion
	test CanApplyMiniumThemeToSite {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
@Brunno0liveira the new test is failing, please check it, thanks!

**Comment by victoroscuro** [comment]
@Brunno0liveira can you please check the test that is failing?

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
> @Brunno0liveira can you please check the test that is failing?

The test was updated, waiting for review

**Comment by lorenzolapescara** [comment]
`CanApplyMiniumThemeToSite` is still failing. We need to investigate what's happening here

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Pull request closed without merging**